src/reservations/dto/reservation-member.dto.ts

```
@@ -0,0 +1,50 @@
+        example: '김파디',
+      })
+      nickname: string;
+    
+      @ApiProperty({
+        description: '프로필 이미지 코드 (01-05)',
+        example: '01',
+        enum: ['01', '02', '03', '04', '05'],
+      })
+      profileImageCode: string;
+    
+      @ApiProperty({
```

ESLint

[error] Delete `····`

(prettier/prettier)